src/sentry/features/handler.py

```diff
@@ -13,6 +16,23 @@
 
 class FeatureHandler:
     features: MutableSet[str] = set()
+    rollout_options: MutableSet[str] = set()
+    option_ttl = 60  # cache the options for 60s
+
+    def __init__(self):
+        self.last_updated = None
+        self.cached_rollouts = {}
+
+    def update_cache_options(self):
+        now = time()
+        if (
+            self.last_updated is None
+            or now - self.option_ttl < self.last_updated  # refetch if the cache is old
+        ):
+
+            self.last_updated = now
+            for rolloutOption in self.rollout_options:
+                self.cached_rollouts[rolloutOption] = options.get(rolloutOption)
```

Comment:
```suggestion
            for rollout_option in self.rollout_options:
                self.cached_rollouts[rollout_option] = options.get(rollout_option)
```

Reply:
I'm thinking this is less and less worth doing generically, @markstory already made `is_in_rollout_group` in the getsentry/feature.py file.